CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - `torch` numeric types are now loaded lazily
 - Reorganized acquisition.py into `acquisition` subpackage
+- Reorganized simulation.py into `simulation` subpackage
```

Comment:
For me, having a general workaround that is "hidden" is fine as long as it is implemented properly. I will give this a proper review once the PR this one depends on is merged and would (as always) highly appreciate a ping from @AdrianSosic once this is the case :)